charts/stable/gollum/templates/_container.tpl

```diff
@@ -0,0 +1,78 @@
+{{- /*
+The main container included in the controller.
+*/ -}}
+{{- define "common.controller.mainContainer" -}}
+- name: {{ include "common.names.fullname" . }}
+  image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+  imagePullPolicy: {{ .Values.image.pullPolicy }}
+  {{- with .Values.command }}
+  {{- if kindIs "string" . }}
+  command: {{ . }}
+  {{- else }}
+  command: 
+  {{ toYaml . | nindent 2 }}
+  {{- end }}
+  {{- end }}
+  - {{ "--ref" }}
+  - {{ .Values.init.gitBranch }}
```

Comment:
Redefining the entirety of common.controller.mainContainer for the inclusion of one item is not the way to go. This is designed to immediately accrue drift.
As you can see from line 8, values.command is already passable and this can be done in a common.yaml by modifying the end resulted .Values